apps/storybook/src/LabeledDropdown.stories.tsx

```diff
@@ -63,10 +92,10 @@ export const Invalid = () => (
       { value: 'LCY', label: 'City' },
       { value: 'SEN', label: 'Southend' },
     ]}
-    value="LHR"
+    value=""
     customValidationMessage={text(
       'Validation Error Message',
-      'This airport is currently closed.',
+      'Choose something',
```

Comment:
```suggestion
      'This airport is currently closed.',
```

Reply:
I don't know why this story has changed. Custom validation message is a server side error. Its misleading to make this error sound like it was caused because the field is empty